test/k8s/manifests/cnp-default-deny-ingress.yaml

```diff
@@ -1,8 +0,0 @@
-apiVersion: "cilium.io/v2"
```

Comment:
This one is still in use here: https://github.com/cilium/cilium/blob/062549799357504bb13ceb1e7854aef56ecf6130/test/k8s/net_policies.go#L1585 and here: https://github.com/cilium/cilium/blob/062549799357504bb13ceb1e7854aef56ecf6130/test/k8s/net_policies.go#L1601